### ISSUE TITLE
SSHD service fails to start

### ISSUE THREAD

**Issue by nkrepo** [PowerShell/Win32-OpenSSH#651]
Please answer the following

**"OpenSSH for Windows" version**
0.0.11.0

**OS details**
Windows 10 Pro Version 1607 Build 14393.953

**What is failing**
The sshd service failed to start due to the following error: 
"The system cannot find the file specified."

I followed installation instructions from [https://github.com/PowerShell/Win32-OpenSSH/wiki/Install-Win32-OpenSSH](url).

The `C:\Program Files\OpenSSH-Win64` is in system PATH and verified that I have installed the SSHD service at `C:\Program Files\OpenSSH-Win64`.

The SSHD service as it was created by the `install-sshd.ps1` :

"PSComputerName          : TESTPC
Name                    : sshd
Status                  : OK
ExitCode                : 0
DesktopInteract         : False
ErrorControl            : Normal
PathName                : C:\Program Files\OpenSSH-Win64\sshd.exe
ServiceType             : Own Process
StartMode               : Manual
__GENUS                 : 2
__CLASS                 : Win32_Service
__SUPERCLASS            : Win32_BaseService
__DYNASTY               : CIM_ManagedSystemElement
__RELPATH               : Win32_Service.Name="sshd"
__PROPERTY_COUNT        : 26
__DERIVATION            : {Win32_BaseService, CIM_Service, CIM_LogicalElement, CIM_ManagedSystemElement}
__SERVER                : TESTPC
__NAMESPACE             : root\cimv2
__PATH                  : \\TESTPC\root\cimv2:Win32_Service.Name="sshd"
AcceptPause             : False
AcceptStop              : False
Caption                 : sshd
CheckPoint              : 0
CreationClassName       : Win32_Service
DelayedAutoStart        : False
Description             : SSH Daemon
DisplayName             : sshd
InstallDate             :
ProcessId               : 0
ServiceSpecificExitCode : 0
Started                 : False
StartName               : NT SERVICE\SSHD
State                   : Stopped
SystemCreationClassName : Win32_ComputerSystem
SystemName              : TESTPC
TagId                   : 0
WaitHint                : 0
Scope                   : System.Management.ManagementScope
Path                    : \\TESTPC\root\cimv2:Win32_Service.Name="sshd"
Options                 : System.Management.ObjectGetOptions
ClassPath               : \\TESTPC\root\cimv2:Win32_Service
Properties              : {AcceptPause, AcceptStop, Caption, CheckPoint...}
SystemProperties        : {__GENUS, __CLASS, __SUPERCLASS, __DYNASTY...}
Qualifiers              : {dynamic, Locale, provider, UUID}
Site                    :
Container               :"

**Comment by manojampalam** [comment]
I'm guessing you don't have the host keys setup. Try doing 
sshd.exe -d
and it should tell you why it wont start

If so, you'll need to do
ssh-keygen.exe -A

**Comment by manojampalam** [comment]
Reopen if needed

**Comment by nkrepo** [comment]
I solved the issue , it was not related with the host keys. I run sshd.exe -d with admin account I managed to connect.
I run the uninstall script and deleted the installation folder.  
Afterwards with an **admin** powershell console I extracted the OpenSSH-Win64.zip as admin as :
```
cd 'C:\Program Files\'
Expand-Archive C:\Users\testuser\Downloads\OpenSSH-Win64.zip ./
mv .\OpenSSH-Win64\ .\OpenSSH
```
and followed the installation guide [https://github.com/PowerShell/Win32-OpenSSH/wiki/Install-Win32-OpenSSH](url)

I am not sure why it didn't work the first time,  the first time I extracted the folder as the "testuser" , copied to `'C:\Program Files\'`.  and afterwards I launched an admin powershell console to proceed with the installation steps.
